cli/src/rest-client/request.ts

```diff
@@ -32,23 +32,29 @@ export async function request(
             console.log(chalk.dim(JSON.stringify(responseBody, null ,2)));
```

Comment:
`if (responseBody)` ?

The whole file looks suspiciously similar to `api.ts`

Reply:
It is deprecated now. Will be removed soon